tools/install_synapse.sh

```diff
@@ -35,7 +35,9 @@ if [[ ! -x ${SYNAPSE} ]]; then
     pushd "${BUILDDIR}"
 
     virtualenv -p "$(which python2)" venv
-    ./venv/bin/pip install "${SYNAPSE_URL}" pyinstaller coincurve pycryptodome 'pysaml2==4.6.0'
+    ./venv/bin/pip install --upgrade pip pyinstaller
```

Comment:
perhaps to avoid problems like that in the future we should pin these dependencies installed via `pip` here?

Reply:
nah, I don't think it's worth the hassle. This can happen anytime an upstream package forgets to include something in the requirements